.github/workflows/build-and-test-pip.yml

```diff
@@ -0,0 +1,77 @@
+name: Build and Test
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  build:
+    name: Build lablink-client
+    runs-on: ubuntu-latest
+
+    permissions:
+      id-token: write
+      contents: read
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+
+    - name: Set up Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: '3.9'
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install --editable .[dev]
+
+    - name: Build package
+      run: |
+        python -m build
+        twine check dist/*
+
+    - name: Upload artifact
+      uses: actions/upload-artifact@v4
+      with:
+        name: lablink-client-dist
+        path: dist/
+        retention-days: 1
+
+  tests:
+    name: Test lablink-client
+    runs-on: ubuntu-latest
+    needs: build
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4

```

Comment:
So we do need the tests folder, but having all the code checked-out can lead to false positive import tests (see [b8005](https://github.com/talmolab/cosyne_vm/pull/5/commits/b8005b74e0eaa3488a4ae2ad0f15e43aecb174d1)). Maybe there is a way to just move the tests folder and change that to our working directory?